### DISCUSSION TITLE
How do I read in my LFS stored csv via pandas?

### DISCUSSION THREAD

**Discussion by bvarnam** [git-lfs/git-lfs#4722] [Q&A]
When I call my pd.read_csv(filepath), I just get 

```
version https://git-lfs.github.com/spec/v1
--
oid sha256:3f0a62d3095514a75196b87b96194a04d95...
size 32868530
```

Where is my actual dataset??? I can't find an answer for this anywhere other than "continue to use normally"; which is frustrating to see.

I understand that it is a pointer file, but I have no idea how to actually use that to get my file.

**Comment by chrisd8088** [comment]
There are a few resources you might want to look at, such as the [tutorial](https://github.com/git-lfs/git-lfs/wiki/Tutorial), which has a [section](https://github.com/git-lfs/git-lfs/wiki/Tutorial#lfs-pointer-files-advanced) on pointer files, and the man [pages](https://github.com/git-lfs/git-lfs/tree/main/docs/man), and there's also a great [tutorial](https://www.atlassian.com/git/tutorials/git-lfs) from Atlassian.

It sounds like you might want to try running a command like `git lfs pull` or `git lfs checkout` if you're seeing a bare Git LFS pointer file in your working tree.

If that's not helpful, I would recommend providing more context in your question; e.g., what is "pandas"?  How did you set up your repository, and specifically the file you are trying to read?  Did you check out a repository that already had Git LFS objects, or did you create a new repository locally?  Do you have the Git LFS client installed on your system, and what kind of system are you using?
